api/RTCIceCandidateStats.json

```diff
@@ -0,0 +1,656 @@
+{
+  "api": {
+    "RTCIceCandidateStats": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCIceCandidateStats",
+        "support": {
+          "chrome": {
+            "version_added": null
```

Comment:
This and other Chromium flavors are false.

Reply:
Fixed in new commit.